### PR TITLE
Add metrics Ansible role

### PR DESCRIPTION
**Pull request by adaszko**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ services:
   prometheus-pushgateway:
     image: prom/pushgateway
     ports:
-      - 19090:9090
+      - 19091:9091
```
Comment (woky) [inline docker-compose.yml]: This will be filtered by firewall on GCE, but otherwise it would be exposed. Better use `127.0.0.1:19091:9091`.

**Review by woky — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (adaszko) [reply docker-compose.yml]: `19091` port is only used on that host, e.g. to get a list of available metrics.  It shouldn't be exposed publicly.

Comment (woky) [reply docker-compose.yml]: But it is. Unless you explicitly specify host IP, listening on `0.0.0.0` aka _all interfaces_ is assumed. Again, this is currently filtered by default firewall rules on GCE. Also, this was there before your commit.

Comment (adaszko) [reply docker-compose.yml]: By publicly, I mean outside of Google Cloud.

Comment (woky) [reply docker-compose.yml]: It'll be exposed outside of Google Cloud if firewall rules on GCE won't filter it out. I'm not sure what's reachable via this port but let's just say this relies on properly configured firewall.

Comment (adaszko) [reply docker-compose.yml]: Changed.

**Pull request merged**